gratipay/models/participant/email.py

```diff
@@ -276,10 +276,20 @@ def finish_package_claims(self, cursor, nonce, *packages):
         """Create teams if needed and associate them with the packages.
         """
         cursor.run('DELETE FROM claims WHERE nonce=%s', (nonce,))
+        if not packages:
+            return
         package_ids = []
+        teams = []
+        team_ids = []
         for package in packages:
-            package.get_or_create_linked_team(cursor, self)
             package_ids.append(package.id)
+            team = package.get_or_create_linked_team(cursor, self)
```

Comment:
@dowski Related to https://github.com/gratipay/gratipay.com/pull/4397#discussion_r112102619, it looks like this is where it becomes useful public API to return a team object from `get_or_create_linked_team`. You okay to bring that back here, to avoid an awkward `package._load_team(cursor)`?